listenbrainz/webserver/static/js/src/recent/RecentListens.tsx

```diff
@@ -202,6 +220,10 @@ export default class RecentListens extends React.Component<
                     artistMBIDs?.length ||
                     Boolean(trackMBID) ||
                     Boolean(releaseGroupMBID);
+
+                  const isListenPersonallyRecommendable = Boolean(
+                    getRecordingMSID(listen)
```

Comment:
Flagging this, but I don't have a direct suggestion:
we're moving aay from using MSIDs as our primary ids for anything, so this feature should be based first on the `recordingMBID`  and only if not available should it be the `recordingMSID`.
Similarly in the API call on the back end we should use MBID first with an optional fallback to MSID

Reply:
I see, this is interesting, I will make changes to the backend and frontend. But I have a question
All the mbid fields in local machine are usually empty. If we do move, development will get hard. I believe we should work on some mechanism to generate dummy MB for development purposes. What do you say?

Reply:
For posterity: there was some discussion on IRC about this, lucifer suggested using an existing script to populate the local DB with an album from MusicBrainz: https://listenbrainz.readthedocs.io/en/latest/developers/commands.html#develop-sh-manage-submit-release

Reply:
Is there a changed required here? I have no visibility on how the whole MBID vs MSID thing is implemented on the back-end so you'll have to tell me :)
Or do we want to change both front-end and back-end at the same time in a separate PR?

Reply:
Since https://github.com/metabrainz/listenbrainz-server/pull/2153, recommendations support both MBID and MSID. Either one of the two is sufficient.

Reply:
Thanks Captain @amCap1712 !
So no change needed then?

Reply:
IIUC, a change is needed to allow for the case when there is only mbid present. Instead of `Boolean(getRecordingMSID(listen))`, something like 
`Boolean(getRecordingMSID(listen)) ||Boolean(getRecordingMBID(listen))` I guess?

Reply:
@hrik2001 We still need the change that lucifer suggested above.
You can use `Boolean(recordingMBID)` since we already use that for `isListenReviewable`